Paccy10/olympus-api
# Bug(properties): fix delete images when deleting property

**What does this PR do?**

- Delete images when deleting a property

**Description of Task to be completed?**

- Delete images on Cloudinary when deleting a property

**How should this be manually tested?**

- Checkout to the branch `bg-delete-images-on-delete-property`
- Run the app with `flask run`
- Go to `postman` and send a `DELETE` request to `/properties{property_id}` and replace `property_id` with the property ID
- The property with the given ID should be deleted and its images should be deleted from Cloudinary

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
